lib/core/controlplane.rb

```diff
@@ -362,4 +363,11 @@ def perform_yaml(cmd)
   def gvc_org
     "--gvc #{gvc} --org #{org}"
   end
+
+  def hide_sensitive_data(message)
+    pattern = @sensitive_data_pattern
+    return message unless pattern.is_a?(Regexp)
```

Comment:
By default, sensitive_data_pattern is `nil`.

The reason I kept it this way was because I had other thoughts. Now I think we can change the line to `return message if pattern.nil?`

Reply:
Oh I see what you mean, I thought you wanted to return the message if there were no matches, but you were just checking for the existence of `@sensitive_data_pattern`.

If we're going to move the pattern to a class constant this won't be relevant anyway, since it will always exist.